### PR TITLE
Do not delete email contact point

### PR DESCRIPTION
**Pull request by pablomendezroyo**
Do not delete email contact point. This way we avoid sending duplicated welcome emails

### CODE REVIEW

File: register/src/grafanaApi/deleteTargetContactPoint.go [hunk 1]
```diff
@@ -8,14 +8,26 @@ import (
...
 // Delete the contact point of a given instance
+// If the contact point is an email, it will not be deleted, this way we avoid sending duplicated welcome emails
 func DeleteTargetContactPoint(instance string, grafanaClient gapi.Client) error {
 	contactPoints, err := grafanaClient.ContactPointsByName(instance)
 	if err != nil {
 		return fmt.Errorf("failed to get contact points by name: " + err.Error())
 	}
-	if len(contactPoints) > 0 {
+	if len(contactPoints) == 1 {
+		if contactPoints[0].Type != "email" {
```
Comment (dsimog01) [inline register/src/grafanaApi/deleteTargetContactPoint.go:18]: Why handling if it is an email? If we implement Telegram or other contact points, we will probably be sending welcome notification the same as with email, so we will likely want to handle it the same way

File: register/src/grafanaApi/deleteTargetContactPoint.go [hunk 2]
```diff
@@ -8,14 +8,26 @@ import (
...
+				return fmt.Errorf("failed to delete contact point: " + err.Error())
+			}
+		} else {
+			logger.Debug("Found email contact point, not deleting")
+		}
+	} else if len(contactPoints) > 1 {
 		for _, contactPoint := range contactPoints {
-			if contactPoint.Name == instance {
+			// This loop should never be executed, but just in case
+			if contactPoint.Name == instance && contactPoint.Type != "email" {
```
Comment (dsimog01) [inline register/src/grafanaApi/deleteTargetContactPoint.go:30]: Same as above

**Review by dsimog01 — APPROVED** [review]

File: register/src/grafanaApi/deleteTargetContactPoint.go (hunk 1 above)
Comment (pablomendezroyo) [reply register/src/grafanaApi/deleteTargetContactPoint.go:18]: yes but we do not have telegram implemented yet

**Comment by pablomendezroyo** [comment]
wont implement

**Pull request closed without merging**